webkit2/jscore.jsc-value.lisp

```diff
@@ -164,3 +164,115 @@
   (value (g-object jsc-value))
   (indent :int))
 (export 'jsc-value-to-json)
+
+(defvar *js-null-value* :null
+  "The value used when translating null from JavaScript to Lisp.")
+(export '*js-null-value*)
+
+(defvar *js-undefined-value* :undefined
+  "The value used when translating undefined from JavaScript to Lisp.")
+(export '*js-undefined-value*)
+
+(defvar *js-false-value* nil
+  "The value used when translating false from JavaScript to Lisp.")
+(export '*js-false-value*)
```

Comment:
The choice between nil for `false` and nil for `null` is delicate.

Any idea what other CL JSON libraries do?

Reply:
The [blogpost you mentioned in #56](https://sabracrolleton.github.io/json-review) lists `nil` as a more widespread value for falsity (5 out of 7 libraries do that), than for nullness (3 or 4 out of 7). Given that using `nil` for both is not an option, it'd better be a boolean value, along with t for truth.